archetype/alloy-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -6,7 +6,7 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.liferay.faces', name: 'com.liferay.faces.alloy', version:'3.0.0-SNAPSHOT'
+    compile group: 'com.liferay.faces', name: 'com.liferay.faces.alloy', version:'3.0.0'
```

Comment:
@vsingleton This *also* should be 2.0.0, please fix here in the 2.x branch PR, squash, and push. Thanks.